### PR TITLE
Added KITE testing development page

### PR DESCRIPTION
**Pull request by namvuCosmo**
Added Testing with KITE tab in development drop-down in nav-bar.

Added KITE testing page, with an iframe referring to a public facing dashboard for KITE tests' results

### CODE REVIEW

File: testing/kite/index.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+---
+
+
+{% include toc-hide.html %}
+
+
+## This is Karoshi Interoperability Testing Engine (KITE)
+
+
+The effortless way to test WebRTC compliance, prevent [Karoshi](https://en.wikipedia.org/wiki/Kar%C5%8Dshi) with KITE!
```
Comment (alvestrand) [inline testing/kite/index.md]: Delete this line. it's fluff.

File: testing/kite/index.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+KITE is designed to be a generic, reusable and easy to maintain automated testing environment.
+
+So far, our developments on KITE allow interoperability testing on stable versions of all major web browsers, namely Chrome, Firefox, Safari and Edge, running on desktop OS, that is either Ubuntu, MacOS or Windows 10.
+
+Mobile testing (Chrome,Firefox on Android, Safari on iOS) is soon to be operational.
```
Comment (alvestrand) [inline testing/kite/index.md]: should say "in development" - never promise a time if you can avoid it :-)

File: testing/kite/index.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+        b) Checks whether both the browsers'iceConnectionState is either 'completed' or 'connected'.
+
+    4) The test is considered as successful if all the browsers either returns 'completed' or 'connected' within 1 minute.
+
+
+### Simple result Dashboard for KITE tests:
+
+
+<iframe width="100%" height="1200" src="https://kiteboard.herokuapp.com/public" frameborder="0" scrolling="no"></iframe>
```
Comment (alvestrand) [inline testing/kite/index.md:43]: is this what we intend to be the permanent home of the dashboard?

File: testing/kite/index.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+Mobile testing (Chrome,Firefox on Android, Safari on iOS) is soon to be operational.
+
+### Development:
+
+Check out KITE on [webrtc/KITE public repository](https://github.com/webrtc/KITE).
+
+### Included sample IceConnectionTest test
+
+IceConnectionTest algorithm is as follows:
```
Comment (alvestrand) [inline testing/kite/index.md]: Suggest adding before this:
"The ICE Connection test is a basic test showing that two browsers can establish a connection."

File: testing/kite/index.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+So far, our developments on KITE allow interoperability testing on stable versions of all major web browsers, namely Chrome, Firefox, Safari and Edge, running on desktop OS, that is either Ubuntu, MacOS or Windows 10.
+
+Mobile testing (Chrome,Firefox on Android, Safari on iOS) is soon to be operational.
+
+### Development:
+
+Check out KITE on [webrtc/KITE public repository](https://github.com/webrtc/KITE).
+
+### Included sample IceConnectionTest test
```
Comment (alvestrand) [inline testing/kite/index.md]: Reads better if you leave off the second "test".

File: testing/kite/index.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+KITE is designed to be a generic, reusable and easy to maintain automated testing environment.
+
+So far, our developments on KITE allow interoperability testing on stable versions of all major web browsers, namely Chrome, Firefox, Safari and Edge, running on desktop OS, that is either Ubuntu, MacOS or Windows 10.
+
+Mobile testing (Chrome,Firefox on Android, Safari on iOS) is soon to be operational.
+
+### Development:
+
+Check out KITE on [webrtc/KITE public repository](https://github.com/webrtc/KITE).
```
Comment (alvestrand) [inline testing/kite/index.md]: "You can check out the source for KITE and run your own tests here:"

**Review by alvestrand — COMMENTED** [review]
A few suggestions for improvement. Huib should be the one to approve, I think.

File: testing/kite/index.md (hunk 3 above)
Comment (namvuCosmo) [reply testing/kite/index.md:43]: Yes.

**Review by alvestrand — APPROVED** [review]
This looks good to me.

**Pull request merged**